packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -129,6 +129,7 @@ export function ListViewItem<T>(props: ListViewItemProps<T>) {
   let {visuallyHiddenProps} = useVisuallyHidden();
 
   let dropProps = isDroppable ? droppableItem?.dropProps : {'aria-hidden': droppableItem?.dropProps['aria-hidden']};
+  let isVirtualDragging = dropHooks?.isVirtualDragging() || dragHooks?.isVirtualDragging();
```

Comment:
So I guess once your other PR goes in, this will end up being merged anyway 😂

Reply:
yep :)